### PR TITLE
form changes: added Placeholder, Labels, CSS

### PR DESCRIPTION
**Pull request by miauzjpg**
added placeholder to inputs, label tags, two column CSS, nicer success and error messages

### CODE REVIEW

**Comment by jayay** [comment]
Your change makes the forms a lot prettier. However, it is unclear to me whether a `placeholder` is wanted for `<input>`s. It makes things look odd to me and I can't be the only one.

image

**Review by jayay — CHANGES_REQUESTED** [review]

File: templates.dist/form/applicantform.php [hunk 1]
```diff
@@ -43,8 +43,19 @@
...
+
+	if ($typeCurrentInput == onOffice\WPlugin\Types\FieldTypes::FIELD_TYPE_BOOLEAN) {
+		$line = "<div class='oo-control'>";
+		$line .= "<label class='oo-control__label' for=".$input.">";
+		$line .= renderFormField($input, $pForm);
+		$line .= $pForm->getFieldLabel($input).$addition."</label>";
+		$line .= "</div>";
+	}
+	else {
+		$line = "<label for=".$input.">". $pForm->getFieldLabel( $input ).$addition.':</label>';
```
[the remark this answers is not in the document]
Comment (miauzjpg) [reply templates.dist/form/applicantform.php]: ok, done.

File: templates.dist/form/applicantform.php [hunk 2]
```diff
@@ -62,19 +73,21 @@
 if ($pForm->getFormStatus() !== \onOffice\WPlugin\FormPost::MESSAGE_SUCCESS) {
 
 ?>
-	<p>
-	<h1>Ihre Kontaktdaten</h1>
+<div class="oo-formfieldwrap">
+
+	<h2>Ihre Kontaktdaten</h2>
```
[the remark this answers is not in the document]
Comment (miauzjpg) [reply templates.dist/form/applicantform.php]: added translation

File: templates.dist/form/defaultform.php [hunk 3]
```diff
@@ -60,8 +60,19 @@
...
+
+		if ($typeCurrentInput == onOffice\WPlugin\Types\FieldTypes::FIELD_TYPE_BOOLEAN) {
+			echo "<div class='oo-control'>";
+			echo "<label class='oo-control__label' for=".$input.">";
+			echo renderFormField($input, $pForm);
+			echo $pForm->getFieldLabel($input).$addition."</label>";
+			echo "</div>";
+		}
+		else {
+			echo "<label for=".$input.">". $pForm->getFieldLabel($input).$addition.": </label>";
```
[the remark this answers is not in the document]
Comment (miauzjpg) [reply templates.dist/form/defaultform.php]: ok, done.

**Comment by miauzjpg** [comment]
> Your change makes the forms a lot prettier. However, it is unclear to me whether a `placeholder` is wanted for `<input>`s. It makes things look odd to me and I can't be the only one.
> 
> image

First we want to display Placeholder and in the next steps maybe hints.

**Comment by jayay** [comment]
If you feel like this should be done, please create a task so that it becomes an official project. It is too risky for me to just accept "random" changes.

**Pull request closed without merging**